server.js

```diff
@@ -1,9 +1,17 @@
 const express = require('express');
 
+const cors = require('cors');
+
 const app = express();
 
+// Then use it before your routes are set up:
```

Comment:
This comment does not describe why you put the line below here. It should be something along the lines of enabling cross origin requests for dev environment or something.